transposon/gene_data.py

```diff
@@ -39,6 +39,8 @@ def __init__(self, gene_dataframe, logger=None):
         self.stops = self.data_frame.Stop.to_numpy(copy=False)
         self.lengths = self.data_frame.Length.to_numpy(copy=False)
         self.chromosomes = self.data_frame.Chromosome.to_numpy(copy=False)
+        self.genome_id = "DEFAULT_GENOME_NAME"
+        # TODO SCOTT pls implement a variable for genome identifier, e.g. camarosa?
```

Comment:
I will use the `g_fname` variable from the bottom of `__main__` in `density.py` for that. Sound good?

Reply:
yes, and in another branch pls